src/GafferImageUI/ImageView.cpp

```diff
@@ -480,6 +478,19 @@ class Box2iGadget : public GafferUI::Gadget
 				return;
 			}
 
+			V2f planarScale = viewportPlanarScale();
+			const ViewportGadget *viewportGadget = ancestor<ViewportGadget>();
+			const ImageGadget *imageGadget = static_cast<const ImageGadget *>( viewportGadget->getPrimaryChild() );
+			// Weird hack to reverse engineer imageGadget->format().getPixelAspect() since it's not public
+			float pixelAspect = 1.0f / imageGadget->pixelAt( LineSegment3f( V3f( 1, 0, 0 ), V3f( 1, 0, 1 ) ) ).x;
+			// Compensate for pixelAspect so UI doesn't get stretched
+			planarScale.x /= pixelAspect;
+
+			const V2f threshold( planarScale * m_handleSize );
```

Comment:
Threshold shouldn't be affected by pixel aspect - it's about distance in screen pixels, not image pixels.